Sources/Rendering/Core/RenderWindow/index.js

```diff
@@ -97,7 +97,9 @@ function vtkRenderWindow(publicAPI, model) {
   publicAPI.getStatistics = () => {
     const results = { propCount: 0, invisiblePropCount: 0, gpuMemoryMB: 0 };
     model._views.forEach((v) => {
-      results.gpuMemoryMB += v.getGraphicsMemoryInfo() / 1e6;
+      // TODO : remove test when 'vtkWebGPURenderWindow.getGraphicsMemoryInfo()' is implemented
+      if (v.getAllocatedGPUMemoryInBytes)
```

Comment:
shouldn't the test be on `v.getGraphicsMemoryInfo` instead

Reply:
did this